Gaijinizer patch/Map229.txt

```diff
@@ -207,17 +235,21 @@ loneliness...Isn't that a product sold regularly?」#MTLed
 それに、あなたの研究しているエジプトのとは違って
 これはヘラのスフィンクスで…！
 > CONTEXT: Map229/events/5/pages/0/161/Dialogue/kazuya
-And it's a special order! Besides, unlike Egypt
-you are studying. Here's Hera's Sphinx ...!#MTLed
+I-it's custom-made!
+
+Besides, it's different from the Egyptian one
+you're researching. This one is Hera's Sphinx...!#Strate
 > END STRING
 
 > BEGIN STRING
 ＊「なんと！
 
 　　ではピキオン山に鎮座したといわれる、あちらの…！」
 > CONTEXT: Map229/events/5/pages/0/169/Dialogue
-*「How! Then there is said to have been settled on Mt.
-Piquion ...!」#MTLed
+*「Oh my!
+   
+   Then it is the one that is said to have been
+   enshrined on Mt. Pikion, that one...!」#Strate
 > END STRING
```

Comment:
We took a look on Discord, I think the English non-romanized version of the mountain the sphinx was on in Oedipus was Mt. Phicium.